spec/emittr_spec.rb

```diff
@@ -101,6 +101,101 @@
     end
   end
 
+  describe '#on_any' do
+    let(:block)     { Proc.new {} }
+    let(:any_block) { Proc.new {} }
```

Comment:
This should be on the context below, since it's not used for when no specific event listeners are set.